app/src/main/res/layout/activity_new_class.xml

```diff
@@ -79,12 +74,9 @@
             <Button
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:text="done"
+                android:text="@string/promt_done"
```

Comment:
You changed the text to "@string/pomt_done" but it seems like you didn't set the "promt_done" in the strings.xml